### DISCUSSION TITLE
DOM manipulation

### DISCUSSION THREAD

**Discussion by gargajit** [fastn-stack/fastn#630] [Ideas & RFCs]
DOM manipulation should be introduced.

Suggested by: @saurabh-lohiya

**Comment by amitu** [comment]
@gargajit, @saurabh-lohiya, I would need more details here. Can you explain what kind of DOM manipulation? Maybe some use case would also help. 

Have you seen the https://fastn.io/blog/web-components/ support we have, does that not work?

**Comment by saurabh-lohiya** [comment]
Since we're able to add and remove elements with `web-components` so that will be fine for now!
